com.valvesoftware.SteamLink.yml

```diff
@@ -0,0 +1,208 @@
+app-id: com.valvesoftware.SteamLink
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+command: steamlink
+
+finish_args:
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/lib
+  - "--env=SDL_GAMECONTROLLERCONFIG_FILE=/var/data/Valve Corporation/SteamLink/controller_map.txt"
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
+
+tags:
+  - proprietary
+  - beta
```

Comment:
Should it be merged to the beta branch or it's a leftover?

Reply:
This is the first ever public release of Steam Link for desktop Linux, so if it's possible to create the repository and release it to a beta branch, in such a way that we can re-release as stable later without needing your intervention, then that would probably be best.

Or if that's not possible, I can remove the `beta` tag and we can go directly to calling it a stable release?

Reply:
For some context on this, Valve has historically released everything on desktop Linux as generic, hopefully-portable binaries built on some suitably old operating system, which we then have to try to support on everyone's favourite distributions, each of which is different. Obviously this doesn't scale very well.

For the Steam Link app, we're trying a different approach where the first public release is already a Flatpak app, and we get the cross-distro portability from Flatpak/Flathub/the fd.o runtime.

Reply:
Yes, we can merge it to the beta branch, which will make it available in `flathub-beta`. When you feel confident to publish it to "stable" Flathub, you just need to create the `master` branch in the git repository.

Reply:
That would be perfect.

Are there any other blockers that I need to fix before this can land as a beta?